vue-frontend/src/components/android-app-development/SuccessStorySection.vue

```diff
@@ -0,0 +1,297 @@
+<template>
+  <section>
+    <div class="div tw-relative tw-mt-10 tw-mb-20">
+      <h2
+        class="tw-pt-5 sm:tw-py-10 tw-text-center tw-text-[1.875rem] lg:tw-text-[3.4375rem] lg:tw-leading-[5.15625rem] md:tw-text-[2.813rem] tw-leading-[2.4375rem] md:tw-leadig-[3.875rem] tw-font-inter-bold tw-text-black-core/[0.87]"
+      >
+        Success Stories
+      </h2>
+      <!-- Mobile UI -->
+
+      <div
+        class="md:tw-hidden tw-container tw-my-[2rem] tw-h-[700px] tw-relative tw-overflow-hidden"
+      >
+        <div
+          class="tw-w-full tw--mt-4 tw-overflow-y-scroll hidden-scrollbar before:tw-z-20 before:tw-blur-lg before:tw-absolute before:tw-left-0 before:tw-h-[8%] before:tw-w-full before:tw-bg-white-gradient"
+        ></div>
+        <div class="swiper-content">
+          <swiper
+            :slidesPerView="1.2"
+            :spaceBetween="20"
+            :direction="'vertical'"
+            :autoplay="{
+              delay: 1000,
+              disableOnInteraction: false,
+            }"
+            :loopedSlides="100"
+            :speed="8000"
+            :loop="true"
+            class="swiper-container tw-h-[700px]"
```

Comment:
success stories section working fine in auto , but when we hover other user image it not working good see.

https://github.com/canopas/canopas-website/assets/115449373/a7be15a8-5c04-4166-a2a6-d237464c34ef